apps/src/templates/rubrics/AiAssessmentBox.jsx

```diff
@@ -136,6 +136,8 @@ export default function AiAssessmentBox({
     );
   };
 
+  const [aiFeedbackId, setAiFeedbackId] = useState(null);
+
   const {aiFeedback, setAiFeedback} = useContext(AiAssessmentFeedbackContext);
```

Comment:
good question! I looked into this a bit because I actually tried to move all that was in AiAssessmentFeedbackContext down into AiAssessmentBox, before realizing that LearningGoals needs to be able to reset it when we switch learning goals. since the higher component (LearningGoals) needs to access this state, AFAIK there's no way to move the state further down, so keeping the state up high (and using a context) makes sense.

for why I used state in `AiAssessmentBox`, my thought was to keep the state in the lowest common ancestor of all the components that need it. However, now I am wondering if this state actually should also be reset when we switch learning goals? I don't currently see a problem, but I think I am just getting away with it because setting the feedback back to `NO_FEEDBACK` is enough to hide the feedback form. 

Conclusion: since I may be hiding a latent bug, and we've already gone through the trouble of extracting `AiAssessmentFeedbackContext`, I think it makes sense to take your suggestion and move this to `AiAssessmentFeedbackContext`. 👍